doc-content/enterprise-only/integration/dev-studio-install-proc.adoc

```diff
@@ -1,26 +1,26 @@
 [id='dev-studio-install-proc']
-= Installing Red Hat JBoss Developer Studio
+= Installing Red Hat Developer Studio
 
-Red Hat JBoss Developer Studio is the JBoss integrated development environment (IDE) based on Eclipse. It integrates tooling and runtime components by combining Eclipse, Eclipse Tooling, and {EAP_LONG}. Red Hat JBoss Developer Studio provides plug-ins with tools and interfaces for {PRODUCT}.
```

Comment:
Change to:
...is an integrated development environment. We probably shouldn't ref JBoss anymore.   Also search for and change Red Hat JBoss Developer Studio to Red Hat Developer Studio. However, Red Hat JBoss Enterprise Application Platform IS correct.

Reply:
Done